src/lib/swapper/swappers/LifiSwapper/endpoints.ts

```diff
@@ -78,11 +76,19 @@ export const lifiApi: Swapper2Api = {
   },
 
   checkTradeStatus: async ({
-    tradeId,
+    quoteId,
+    txHash,
```

Comment:
Feels like this should belong in a separate PR since this is not CoW-related, probably in #4882, though since this is related to the new swapper under flag, happy to keep it here

Reply:
thank you ser